### PR TITLE
Section Group data scripts

### PR DESCRIPTION
**Pull request by georgend**
Add Data Scripts for Section Group Data. 
Add correct identifiers to identifiers file. 
Remove fixed issues from Known Issues.

### CODE REVIEW

**Comment by zengin** [comment]
@georgend thanks for allowing me to iterate on your PR for the PR checks. GitHub doesn't kick off subsequent workflows even with a PAT. I raised a ticket. Based on their response, I will retract all the experimental commits here.

cc: @fey101 FYI..

File: .github/workflows/create-reports.yml [hunk 1]
```diff
@@ -40,21 +40,13 @@ jobs:
...
\ No newline at end of file
+      uses: zengin/commit@v3.0.2
+      with:
+        directory: ${{ env.BUILD_SOURCESDIRECTORY }}/msgraph-sdk-raptor
+        token: ${{ secrets.PAT }}
+        branch: ${{ env.CURRENT_BRANCH }}
+        message: 'Update reports'
+        force: 'true'
+        name: 'Mustafa Zengin'
+        email: 'muzengin@microsoft.com'
```
Comment (nikithauc) [inline .github/workflows/create-reports.yml:52]: Is it okay that this email is public?

File: .github/workflows/create-reports.yml [hunk 2]
```diff
@@ -40,21 +40,13 @@ jobs:
...
-        git remote add origin https://$env:PAT@github.com/microsoftgraph/msgraph-sdk-raptor.git
-
-        git push origin $env:CURRENT_BRANCH
-      shell: pwsh
-      working-directory: ${{ env.BUILD_SOURCESDIRECTORY }}/msgraph-sdk-raptor
-      env:
-        CURRENT_BRANCH: ${{ env.CURRENT_BRANCH }}
-        PAT: ${{ secrets.PAT }}
\ No newline at end of file
+      uses: zengin/commit@v3.0.2
```
Comment (nikithauc) [inline .github/workflows/create-reports.yml:44]: What does this do?

Comment (zengin) [reply .github/workflows/create-reports.yml:44]: It was addressing the two issues I opened here in a fork:

https://github.com/actions-x/commit/issues/2
https://github.com/actions-x/commit/issues/3

But @RikudouSage is so kind to take an action really quick to fix these, so I will update the reference once the PRs are closed.

File: .github/workflows/create-reports.yml (hunk 1 above)
Comment (zengin) [reply .github/workflows/create-reports.yml:52]: already public in the git history :)

**Comment by zengin** [comment]
woot! Suggestion from GitHub support worked and UnitTests are run on a commit from GitHub Actions.

It looks like I need to remove "create-reports" from required checks though as it won't always be run.

@georgend could you please address the PS linting errors in this PR?

**Comment by georgend** [comment]
@zengin issue fixed.

**Pull request merged**